go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -2050,12 +2053,15 @@ func (tsv *TabletServer) GetTxPoolWaiterCap() int64 {
 // and also truncates data if it's too long
```

Comment:
This comment is not correct, there is no truncate.

Also, I would document that bindVariables=nil has a different meaning than len(bindVariables]==0, as it's not common to do so. And the caller depends on this behavior.